Applications/Spire/Source/KeyBindings/OrderTasksRow.cpp

```diff
@@ -469,10 +475,8 @@ EditableBox* OrderTasksRow::make_editor(
     return new EditableBox(*input_box);;
 }
 
-EditableBox* OrderTasksRow::make_empty_editor(
-    const std::shared_ptr<ComboBox::QueryModel>& region_query_model,
-    const DestinationDatabase& destinations, const MarketDatabase& markets,
-    const std::shared_ptr<TableModel>& table, int row, int column) {
+EditableBox* OrderTasksRow::make_empty_editor(int column,
```

Comment:
What mutable operation does this perform?

Reply:
`on_submit` which is a mutable operation is used for the submit signal of the input box within `make_empty_editor`.   Since 'make_editor' doesn't preform any mutable operation, it should be a const member function.